lib/airbrake/resque/failure.rb

```diff
@@ -7,13 +7,20 @@ module Failure
     # @see https://github.com/resque/resque/wiki/Failure-Backends
     class Airbrake < Base
       def save
+        return if activejob_exception?
         params = payload.merge(
           component: 'resque',
           action: payload['class'].to_s
         )
 
         ::Airbrake.notify_sync(exception, params)
       end
+
+      # AirBrake gem already attaches error notifications to ActiveJob
+      # so it needs to be skipped or duplicate exception will be logged
+      def activejob_exception?
+        payload['class'].to_s == 'ActiveJob::QueueAdapters::ResqueAdapter::JobWrapper'
```

Comment:
I feel like using [`Module#name`](https://ruby-doc.org/core-2.3.1/Module.html#method-i-name) is a bit safer.